### PR TITLE
Support api endpoints

### PR DESCRIPTION
**Pull request by migueljteixeira**
What was added:
(Movies)
- alternative titles
- credits
- images
- keywords
- releases
- videos
- latest movie added
- lists
- reviews

I intend to update the other services in the following days.
Miguel

### CODE REVIEW

**Comment by UweTrottmann** [comment]
Thanks for the PR!

Before I can merge this you need to clean it up though:

- rebase your commits against the latest changes in `dev`, e.g. see [rebaseandsqua.sh](http://rebaseandsqua.sh/).
- use spaces instead of tabs for indentation
- add new lines at the end of source files (so git won't track appending to a file as a change to the last line)

**Comment by migueljteixeira** [comment]
One question: should i keep the "trailers" method on moviesService? It was removed from the api documentation..

**Comment by UweTrottmann** [comment]
Looks like `trailers` was replaced by `videos`. If you add the new `videos` method, you may remove `trailers`, sure.

And yes, you can make multiple commits if it makes sense. However, it might make sense to just open another PR if there are a lot of code changes.

**Comment by migueljteixeira** [comment]
Ok, i pushed another commit, with the changes that you specified.
I intend to update the other services in the following days, may i rebase the commits after that?

**Comment by UweTrottmann** [comment]
Sure. As I said I won't merge until you rebase onto the latest changes. Let me know once you are done!

**Comment by migueljteixeira** [comment]
Thanks! ;)

**Comment by migueljteixeira** [comment]
Hey.
I have updated the services. Can you confirm that is everything alright?

File: src/test/java/com/uwetrottmann/tmdb/services/TvServiceTest.java [hunk 1]
```diff
@@ -13,38 +24,196 @@
...
+        assertThat(videos.results.get(0).name).isNotNull();
+        assertThat(videos.results.get(0).site).isEqualTo("YouTube");
+        assertThat(videos.results.get(0).size).isNotNull();
+        assertThat(videos.results.get(0).type).isEqualTo("Opening Credits");
+    }
+    
+    @Test
+    public void test_latest() {
+        TvShowComplete show = getManager().tvService().latest();
+        assertTvShow(show);
```
Comment (UweTrottmann) [inline src/test/java/com/uwetrottmann/tmdb/services/TvServiceTest.java]: Can't be sure what is returned there (e.g. `poster_path` or other non-essential stuff is `null`), so probably best if you just test for the essential things (like there is a TMDB id).

**Comment by UweTrottmann** [comment]
The test for latest TV show (see my comment) fails if there isn't a `poster_path`, otherwise looks good.

If you know fetch the latest changes from this repositories `dev` branch and rebase on top of it, there is nothing standing in the way of merging this. Thanks again for the hard work!

**Comment by UweTrottmann** [comment]
You know what, forget it. I'll just make this tiny change and merge this for you right now :D :+1:

**Comment by UweTrottmann** [comment]
**Merged**

(I did a rebase onto the `dev` branch and skipped all the unrequired commits not from you, hence GitHub thinks this PR is not merged even if your commits are.)

**Pull request closed without merging**